policy/modules/services/seatd.te

```diff
@@ -0,0 +1,37 @@
+policy_module(seatd)
+
+########################################
+#
+# Declarations
+#
+
+type seatd_t;
+type seatd_exec_t;
+init_daemon_domain(seatd_t, seatd_exec_t)
+
+type seatd_initrc_exec_t;
+init_script_file(seatd_initrc_exec_t)
+
+type seatd_runtime_t;
+files_runtime_file(seatd_runtime_t)
+
+########################################
+#
+# Local policy
+#
+
+allow seatd_t self:capability { chown dac_override sys_admin sys_tty_config };
+allow seatd_t self:unix_stream_socket { accept listen };
+
+manage_files_pattern(seatd_t, seatd_runtime_t, seatd_runtime_t)
+files_runtime_filetrans(seatd_t, seatd_runtime_t, file)
+
+auth_use_nsswitch(seatd_t)
+
+dev_getattr_dri_dev(seatd_t)
+dev_getattr_input_dev(seatd_t)
+dev_rw_dri(seatd_t)
+dev_rw_input_dev(seatd_t)
+
+# seatd may not have the same level as a client
+mls_fd_share_all_levels(seatd_t)
```

Comment:
Usually MLS rules go right after the declaration of the type. Take a look at `podman.te` for example.

Reply:
Ah okay, thanks!